### DISCUSSION TITLE
How can I pull new commits to the created PR?

### DISCUSSION THREAD

**Discussion by sanchezzzhak** [community/community#22798] [New to GitHub]
<p>Previously when adding a commit to a branch the commit was automatically added to the PR</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/sanchezzzhak">@sanchezzzhak</a>! :wave: Welcome to the Community!</p>
<p>I don’t believe this behavior has changed recently? Are you making changes to a branch in a repository you own, and pushing them up to the version on GitHub? Those commits should get added to the PR automatically.</p>
<p>If you’re pushing to a branch in a fork of your repo, however, it could be that the person that opened the PR is not allowing anyone to push to their branch?</p>
<p><a href="https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork" class="onebox" target="_blank">https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork</a></p>

**Comment by sanchezzzhak** [comment]
<p>Everything is fine, after an hour or more of time, my commits were added.</p>
<p>The issue is closed, false alarm.</p>
